package-lock.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "monsters-of-chopa-sipa",
+  "name": "ts-tg-bot",
   "version": "1.0.0",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
-      "name": "monsters-of-chopa-sipa",
+      "name": "ts-tg-bot",
```

Comment:
It changed by some reason

Q: Is that related to some GH template kung fu?

Reply:
@s373r nope, it's because I was making a template from a new project :)